internal/pkg/model/manifest.go

```diff
@@ -51,17 +51,6 @@ type RecordState struct {
 	Deleted   bool // record has been deleted in this command run
 }
 
-type PathInProject struct {
```

Comment:
1. commit, premenovana struktura `PathInProject` na `AbsPath`.
Kedze sa bude pouzivat aj v template, a tam to vyzera divne `InProject`.

+ presunute do suboru `path.go`, v rovnakom balicku `model`.